test/python/read_hippo_data.py

```diff
@@ -1,18 +1,75 @@
+"""Helper functions for reading OpenFOAM and MOOSE exodus data
+"""
+
+import os
+import re
 
 from pathlib import Path
 from typing import Literal
+
 import numpy as np
 import pyvista as pv
 
-def get_exodus_times(exo_file):
+def get_exodus_times(exo_file: Path | str | bytes) -> list[float]:
+    """Get the times from the exodus file.
+
+    Parameters
+    ----------
+    exo_file : Path | str | bytes
```

Comment:
To save time in the future, I'd suggest not putting types in docstrings if you're using type hints.